documentation/source/development/platforms.rst

```diff
@@ -0,0 +1,31 @@
+Supported platforms
+===================
+
+- Vulkan API is completely platform-agnostic, which allows it to run on various operating systems.
+- The required drivers for Vulkan are usually part of your graphic card's drivers.
+- Update your graphics drivers ase often as possible since new drivers with Vulkan updates are released frequently.
+- Driver updates contain new features, bug fixes, and performance improvements.
+- Check out `Khronos website <https://www.khronos.org/vulkan/>`__ for more information.
+
+Microsoft Windows
+-----------------
+
+- We support x64 Microsoft Windows 7, 8, 8.1 and 10.
```

Comment:
Windows 7?
Windows has reached [end of life](https://www.microsoft.com/en-us/microsoft-365/windows/end-of-windows-7-support)

Maybe add it under the category (might work), but not official support.

Reply:
Ok let's remove it.

Reply:
Done.